scripts/deploy-helm.sh

```diff
@@ -286,3 +286,14 @@ fi
 if [[ $DEPLOYMENT_TYPE == metrics ]] || [[ $DEPLOYMENT_TYPE == all ]]; then
   echo "kubectl -n dynatrace logs -l app=dynatrace-gcp-function -c dynatrace-gcp-function-metrics"  > ${CMD_OUT_PIPE}
 fi
+
+if [[ $DEPLOYMENT_TYPE == all ]]; then
+  LOG_MONITORING_URL="${DYNATRACE_URL}/ui/log-monitoring?query=cloud.provider%3D%22gcp%22"
+  LOG_VIEWER="Log Viewer: $LOG_MONITORING_URL"
+fi
+
+if [[ $DEPLOYMENT_TYPE == logs ]] || [[ $DEPLOYMENT_TYPE == all ]]; then
+  echo
+  echo "- Check logs in Dynatrace in 5 min. ${LOG_VIEWER}" >${CMD_OUT_PIPE}
```

Comment:
if you will have only `logs` then variable `LOG_VIEWER` will be not exists here

Reply:
explained

Reply:
Yes, we can show the link only when DYNATRACE_URL is set (for 'all' deployment type), for 'logs' deployment type it's empty deliberately